pages/guides/references/changelog.vue

```diff
@@ -151,7 +151,7 @@ export default {
           <Footer />
         </article>
       </div>
-      <TableOfContents :toc="tableOfContents" :has-banner="Boolean(banner)" />
+      <TableOfContents :toc="tableOfContents" :has-banner="banner" />
```

Comment:
Saw this when I was debugging -- you don't need to use `Boolean` in your templates.